src/global_launch/config/README.md

```diff
@@ -43,9 +43,27 @@ ROS parameters specific to the nodes in the local_pathfinding package.
 **`interval_spacing`**
 
 - _Description_: The upper bound on spacing between each point in the global path in km.
-- _Datatype_: `float`
+- _Datatype_: `double`
 - _Range_: `(0.0, MAX_DOUBLE)`
 
+**`write`**
+
+- _Description_: Whether or not to write a generated global path to a new csv file.
+- _Datatype_: `boolean`
+- _Acceptable Values_: `true`, `false`
+
+**`gps_threshold`**
```

Comment:
Description and acceptable values need to be updated, as well as a log message in node_mock_global_path.py